django_prometheus/middleware.py

```diff
@@ -1,6 +1,9 @@
 from prometheus_client import Counter, Histogram
 from django_prometheus.utils import Time, TimeSince, PowersOf
 import django
+from django.conf import settings
+
+LATENCY_BUCKETS = getattr(settings, "PROMETHEUS_LATENCY_BUCKETS", None)
```

Comment:
how about doing
if LATENCY_BUCKETS is None:
  LATENCY_BUCKETS = DEFAULT_LATENCY_BUCKETS

here, which would simplify the code below?

Reply:
Had thought about that, but wasn't sure what DEFAULTS to provide. I think it probably is fine to use the same defaults as in the prometheus client.